<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutDialog</class>
 <widget class="QDialog" name="CheckoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CHECK OUT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Payment Options</string>
   </property>
   <widget class="QRadioButton" name="credit_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Card</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="paymaya_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Paymaya</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="gcash_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>G-Cash</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cash_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cash</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Total</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="rate_tb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="placeholderText">
     <string>00000</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QRadioButton" name="downpayment_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Downpayment</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fullypaid_rb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fully Paid</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Reciept</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="totalamount_tb">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="placeholderText">
     <string>00000</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="reserve_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reserve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[status]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
